integrationtests/Paket.IntegrationTests/RestoreSpecs.fs

```diff
@@ -10,16 +10,16 @@ open FsUnit
 let ``#2496 Paket fails on projects that target multiple frameworks``() = 
     let project = "EmptyTarget"
     let scenario = "i002496-empty-target-framework"
-    let tmpPaketFolder = (scenarioTempPath scenario) @@ "../.paket"
+    let tmpPaketFolder = (scenarioTempPath scenario) @@ ".paket"
     let targetsFile = FullName(__SOURCE_DIRECTORY__ + "../../../src/Paket/embedded/Paket.Restore.targets")
-    let paketExe = FullName(__SOURCE_DIRECTORY__ + "../../../.paket/paket.exe")
```

Comment:
This is now magic mode and magic mode downloaded the latest stable instead of testing the current build :/

Reply:
So it basically tested the current targets file with an older release. @forki Do we really need to maintain that kind of compat as well?